src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -44,48 +44,48 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Can you also add a javadoc to the class to indicate this servlet is used to update the database w/ most recent election info?

Reply:
Yes, doing so right now!